app/presenters/reservation_calendar.rb

```diff
@@ -0,0 +1,19 @@
+class ReservationCalendar < SimpleDelegator
+
+  def initialize(reservation)
+    super(reservation)
```

Comment:
If we're not doing anything special in the constructor, I think we don't need it.